ui/narrative/methods/run_metaSPAdes/spec.json

```diff
@@ -58,7 +58,8 @@
                 },
                 {
                     "input_parameter": "read_libraries",
-                    "target_property": "read_libraries"
+                    "target_property": "read_libraries",
+                    "target_type_transform": "list<ref>"
```

Comment:
My understanding is that we want the module to accept multiple files in general, and we want to restrict the narrative interface to accepting a single file.